dashboard/src/components/metrics/common/UserTagChart.vue

```diff
@@ -14,26 +14,34 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 <template>
-  <metrics-card :title="title" data-cy="userTagChart">
-    <metrics-overlay :loading="isLoading" :has-data="!isEmpty" no-data-icon="fa fa-info-circle" no-data-msg="No user data yet...">
-      <apexchart :type="this.chartType" height="350"  :options="chartOptions" :series="series"></apexchart>
-    </metrics-overlay>
+  <metrics-card :title="titleInternal" data-cy="userTagChart">
+    <skills-spinner :is-loading="isLoading" class="mb-5"/>
+    <div v-if="!isLoading">
```

Comment:
@sudo-may did you mean to have this `v-if` here?   not sure you need it since the chart is within the `<metrics-overlay>` component which also is using `isLoading`

Reply:
@rmmayo I think it's fine, alternatively it can just say ``:loading="false"`` on the ``<metrics-overlay>``; primary function of the overlay is to show the "no data yet.." message